src/Zlib.xml

```diff
@@ -0,0 +1,35 @@
+<SPDX name="zlib License" identifier="Zlib" osi-approved="true">
+  <urls>
+    <url>http://www.zlib.net/zlib_license.html</url>
+    <url>http://www.opensource.org/licenses/Zlib</url>
+  </urls>
+  <license>
+    <title>
+      <p>zlib License</p>
+    </title>
+    <body>
+      <p>This software is provided 'as-is', without any express or implied warranty. In no event will the authors
+         be held liable for any damages arising from the use of this software.</p>
+      <p>Permission is granted to anyone to use this software for any purpose, including commercial applications,
+         and to alter it and redistribute it freely, subject to the following restrictions:</p>
+      <list></list>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>The origin of this software must not be misrepresented; you must not claim that you wrote the
+             original software. If you use this software in a product, an acknowledgment in the product
+             documentation would be appreciated but is not required.</p>
```

Comment:
Agreed; I think we should think of the has-acknowledgement label as "requires acknowledgement," so we wouldn't attach it where acknowledgement is not required.

I would actually propose changing the label to "requires acknowledgment" for clarity; if people agree I think we would have to select all the PRs with the old label, add the new label, then remove the old.